README.md

```diff
@@ -46,6 +46,36 @@ You can implement your own cache strategy by creating a new class that implement
 
 You can customize the way you handle the users in your application by creating your own `UserRepository`. This class should implement the `Auth0\Login\Contract\Auth0UserRepository` contract. Please see the [Custom User Handling section of the Laravel Quickstart](https://auth0.com/docs/quickstart/webapp/laravel#optional-custom-user-handling) for the latest example.
 
+### Using auth guard
+
+To use the auth0 guard add it to the config/auth.php with the driver "auth0":
+```
+'guards' => [
+    ...
+    'auth0' => [
```

Comment:
the name of the guard can be anything. only the driver is important. the provider has to be the name of the provider defined below